common/src/main/java/net/dodogang/plume/block/TallCeilingPlantBlock.java

```diff
@@ -45,19 +45,28 @@ protected void appendProperties(StateManager.Builder<Block, BlockState> builder)
     }
 
     @Override
-    public BlockState getStateForNeighborUpdate(BlockState state, Direction direction, BlockState newState, WorldAccess world, BlockPos pos, BlockPos posFrom) {
+    public BlockState getStateForNeighborUpdate(
+        BlockState state, Direction direction, BlockState newState, WorldAccess world, BlockPos pos, BlockPos posFrom) {
         DoubleBlockHalf doubleBlockHalf = state.get(HALF);
-        if (direction.getAxis() == Direction.Axis.Y && doubleBlockHalf == DoubleBlockHalf.LOWER == (direction == Direction.UP) && (!newState.isOf(this) || newState.get(HALF) == doubleBlockHalf)) {
+        if (direction.getAxis() == Direction.Axis.Y
+            && doubleBlockHalf == DoubleBlockHalf.LOWER == (direction == Direction.UP)
+            && (!newState.isOf(this) || newState.get(HALF) == doubleBlockHalf))
+        {
```

Comment:
I like this way of doing multi-line if statements

Reply:
I can be indifferent but ehhh

[idea](https://www.jetbrains.com/idea/) posted in discord:
```java
@Override
public BlockState getStateForNeighborUpdate(BlockState state, Direction dir, BlockState newState, WorldAccess world, BlockPos pos, BlockPos posFrom) {
    DoubleBlockHalf half = state.get(HALF);
    if (dir.getAxis() == Direction.Axis.Y && (half == DoubleBlockHalf.LOWER && dir == Direction.UP) && (!newState.isOf(this) || newState.get(HALF) == half)) {
        return Blocks.AIR.getDefaultState();
    } else {
        return half == DoubleBlockHalf.LOWER && dir == Direction.DOWN && !state.canPlaceAt(world, pos)
            ? Blocks.AIR.getDefaultState()
            : super.getStateForNeighborUpdate(state, dir, newState, world, pos, posFrom);
    }
}
```